CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 2021.6.9999 (23 June 2021)
```

Comment:
Maybe the 9999 is silly here? Just following the structure until we get to our next major release and we can drop the extra 9999s and get back on the month schedule.